### DISCUSSION TITLE
How to configure esbuild

### DISCUSSION THREAD

**Discussion by imcm7** [nuxt/nuxt#23332] [Questions]
### Environment

nuxt 3

### Reproduction

-

### Describe the bug

Where to add that https://mikro-orm.io/docs/deployment#excluding-dependencies-from-esbuild
in `vite: { esbuild: { external: ` or `nitro: {   esbuild: { external: } `.

**Comment by imcm7** [comment]
In `vite: { esbuild: { external:` I get error.

**Comment by MrEraxd** [comment]
Hi @imcm7 I don't think this is an Nuxt issue, better place would be to ask in Vite repo: https://github.com/vitejs/vite/discussions.
